src/main/java/com/avengers/gamera/entity/Article.java

```diff
@@ -0,0 +1,65 @@
+package com.avengers.gamera.entity;
+
+import com.avengers.gamera.constant.ArticleType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
```

Comment:
Don't use wild cards import, only import what is needed.

Reply:
fix.  The IDEA always converts to wild card imports event I changed the settings. so annoying